libcontainer/cgroups/utils.go

```diff
@@ -514,8 +514,8 @@ func getHugePageSizeFromFilenames(fileNames []string) ([]string, error) {
 }
 
 // GetPids returns all pids, that were added to cgroup at path.
-func GetPids(path string) ([]int, error) {
-	return readProcsFile(path)
+func GetPids(dir string) ([]int, error) {
```

Comment:
breaks compatibility if there is non-runc consumer of this function

Reply:
Nope (I just changed the name of the parameter to be clear it's a directory, otherwise it's doing the same thing).